Makefile

```diff
@@ -390,23 +388,28 @@ repository-test:
 
 # Main distribution
 
+check-repo: doc/CHANGES
+	git status -uno
+
 # The staging target builds all of the files that will be distributed
 # to the website in the directory $(STAGING_DIR).  This includes:
 # daikon.tar.gz, daikon.zip, daikon.jar, javadoc, and the documentation.
 # See the dist target for moving these files to the website.
-staging: doc/CHANGES
+staging:
 # Our intention is that members of the plse_www group will always have
 # write permission on the release directories; however, if you happen
 # to be the owner of an existing file, the permissions system gives
 # that priority and we must set the write bit.  These two chmod commands
 # will fail if you are not the owner, but the remainder of the commands
 # should work fine.
+	-chmod u+w $(WWW_PARENT)
 	-chmod -R u+w $(STAGING_DIR)
-	-chmod u+w $(WWW_DIR)
 	/bin/rm -rf $(STAGING_DIR)
-	# dummy history directory to remove checklink warnings
 	install -d $(STAGING_DIR)/history
```

Comment:
OK, I see.  Would "mkdir $STAGING_DIR" be enough?  That is, does this need to create the history/ subdirectory?

Also, maybe it's just me but I find using "install" counterintuitive when just creating a directory.  (That's what is happening here, right?)  I would use "mkdir" for that, because when I see "install" I assume it is copying a directory.